jplag.endToEndTesting/src/main/java/de/jplag/end_to_end_testing/modelRecord/ExpectedResult.java

```diff
@@ -0,0 +1,7 @@
+package de.jplag.end_to_end_testing.modelRecord;
+
+import com.fasterxml.jackson.annotation.JsonProperty;
+
+public record ExpectedResult(@JsonProperty("minimal_similarity") float resultSimilarityMinimum,
```

Comment:
Records are still public types and thus need JavaDoc!

Reply:
added comments